packages/footer/components/Navigation.tsx

```diff
@@ -10,9 +10,90 @@ import {
    ListProps,
    SimpleGrid,
    SimpleGridProps,
+   Text,
 } from '@chakra-ui/react';
+import { NewsletterComponent, NewsletterFormProps } from './Newsletter';
+import { Menu as MenuType, MenuItem } from '../../ui/menu/menu';
```

Comment:
Oh... this is possibly a problem. I think we should be using the `@ifixit/ui` alias here instead of a relative path that goes *outside* this package directory.

Reply:
[32deab5](https://github.com/iFixit/react-commerce/pull/1342/commits/32deab58142a96fc9145cf6932e6a8d825ab2574)